packages/features/flags/server/utils.ts

```diff
@@ -0,0 +1,13 @@
+import type { PrismaClient } from "@prisma/client";
+
+import type { AppFlags } from "../config";
+
+export async function getFeatureFlagMap(prisma: PrismaClient) {
+  const flags = await prisma.feature.findMany({
+    orderBy: { slug: "asc" },
+  });
+  return flags.reduce<AppFlags>((fs, f) => {
+    fs[f.slug as keyof AppFlags] = f.enabled;
+    return fs;
+  }, {} as unknown as AppFlags);
```

Comment:
Do we need this `unknown` seems to work fine for me 

```suggestion
  return flags.reduce<AppFlags>((acc, flag) => {
    acc[flag.slug as keyof AppFlags] = flag.enabled;
    return acc;
  }, {} as AppFlags);
```